### DISCUSSION TITLE
Fail to boot on  AMD Radeon Rx 6750 Gre 12GB  Device ID  1002 73DF-1EAE 6730

### DISCUSSION THREAD

**Discussion by ghost** [ChefKissInc/NootRX#68] [General]
Fail to boot on  AMD Radeon Rx 6750 Gre 12GB  Device ID  1002 73DF-1EAE 6730

**Comment by z3r0c00l-2k** [comment]
Try to build the NootRX.kext from the latest source.
I had issues with booting with NootRX.kext enabled. So I built from the latest source and it worked.

**Comment by eeshankeni** [comment]
Disable NootRX in config.plist before installation, took me way too long to figure this out. Ought to be in the docs...
